packages/fx-core/src/component/driver/aad/update.ts

```diff
@@ -100,6 +101,19 @@ export class UpdateAadAppDriver implements StepDriver {
         };
       }
       if (axios.isAxiosError(error)) {
+        if (
+          error.response!.status == 400 &&
+          error.response!.data.error.code === aadErrorCode.permissionErrorCode
+        ) {
+          const message = JSON.stringify(error.response!.data.error.message);
```

Comment:
`error.aad.manifest.AADPermissionIssues` is not appended to the raw error in response.

Reply:
updated.